packages/drivekit_core/flutter_drivekit_core/example/android/app/build.gradle

```diff
@@ -34,6 +34,7 @@ android {
     }
 
 
+    namespace = "com.example.verygoodcore.example"
```

Comment:
Those package names come from the CLI but it should be something like com.drivekit.core (and maybe add .flutter if you think it could create a conflict)

Reply:
Yes exactly.
We will do it in a future PR. I've created a ticket in kanban
https://www.notion.so/m33/Set-the-sdk-ID-10cbc74dd51242fa9b0257bb3e867b1b?pvs=4